src/widgets/breadcrumb/breadcrumb.tsx

```diff
@@ -113,7 +113,7 @@ export type BreadcrumbWidgetParams = {
 };
 
 export type BreadcrumbWidget = WidgetFactory<
-  BreadcrumbRendererOptions,
+  BreadcrumbWidgetDescription & { $$widgetType: 'ais.breadcrumb' },
```

Comment:
is the `{ $$widgetType: 'ais.breadcrumb' }` part need?
It seems it's already inside [`BreadcrumbWidgetDescription`](https://github.com/algolia/instantsearch.js/blob/70715e09d078e556552aab357d5ff3cf92154232/src/connectors/breadcrumb/connectBreadcrumb.ts#L80)

Reply:
yes, otherwise it's a connector, not a widget :)